src/App/PhotosList/VirtualPhotoGrid.jsx

```
@@ -107,6 +111,21 @@ function VirtualPhotoGrid({
     containerRef: _containerRef
 }) {
     const gridRef = useRef(null);
+
+    // Stabilize the callbacks handed to cells. The upstream handlers (e.g.
+    // addSelection) close over selection state and get a new identity on every
+    // toggle; if passed straight into cellProps they'd change its identity and
+    // force react-window to re-render every cell. Wrapping them in refs keeps
+    // cellProps referentially stable while still invoking the latest logic.
+    const onAddSelectionRef = useRef(onAddSelection);
+    onAddSelectionRef.current = onAddSelection;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

  119 |     // cellProps referentially stable while still invoking the latest logic.
  120 |     const onAddSelectionRef = useRef(onAddSelection);
> 121 |     onAddSelectionRef.current = onAddSelection;
      |     ^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot update ref during render
  122 |     const onDisplayPhotoRef = useRef(onDisplayPhoto);
  123 |     onDisplayPhotoRef.current = onDisplayPhoto;
  124 |     const onOpenBurstGroupRef = useRef(onOpenBurstGroup);

(react-hooks/refs)